components/userMenu.tsx

```
@@ -0,0 +1,13 @@
+"use client";
+
+import { signOut } from "@/lib/firebase/auth";
+import Link from "next/link";
+
+export default function UserMenu({ username }: { username: String }) {
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)